store/tikv/gcworker/gc_worker.go

```diff
@@ -258,11 +258,26 @@ func (w *GCWorker) leaderTick(ctx context.Context) error {
 		return nil
 	}
 
+	stores, err := w.getUpStores(ctx)
+	concurrency := len(stores)
+	if err != nil {
+		logutil.Logger(ctx).Error("[gc worker] failed to get up stores to calculate concurrency",
+			zap.String("uuid", w.uuid),
+			zap.Error(err))
+		concurrency = gcDefaultConcurrency
```

Comment:
How about load from mysql.tidb first here, and then use default value?

Reply:
Why is it necessary?